datafusion/sqllogictest/test_files/spark/string/luhn_check.slt

```diff
@@ -15,23 +15,134 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# This file was originally created by a porting script from:
```

Comment:
Can we please leave the identifying comments (specifically the ORIGINAL query and the PySpark 3.5 result lines)?